CMakeLists.txt

```diff
@@ -159,6 +159,9 @@ add_executable(nothing
 )
 target_link_libraries(nothing ${SDL2_LIBRARIES} system)
 
+ADD_CUSTOM_TARGET(link_assets ALL
+                  COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/assets ${CMAKE_BINARY_DIR}/assets)
```

Comment:
Using symlinks for a crossplatform application is not a good idea. It is not very conventional on some of the platforms. Especially the most popular ones.